### PR TITLE
ExpGenerateProposals: Added tmpBuffer

### PR DESCRIPTION
**Pull request by IvanPoliksenov**
Added tmpBuffer

### CODE REVIEW

File: inference-engine/src/vpu/graph_transformer/src/stages/exp_generateproposals.cpp [hunk 1]
```diff
@@ -129,6 +131,13 @@ void FrontEnd::parseExpGenerateProposals(
         outputs);
 
     stage->attrs().set("params", params);
+
+    const int numProposals = sizeof(uint8_t) * 2 * inputScores->desc().dim(Dim::H) * inputScores->desc().dim(Dim::W) * inputScores->desc().dim(Dim::C);
```
Comment (Maxim-Doronin) [inline inference-engine/src/vpu/graph_transformer/src/stages/exp_generateproposals.cpp:135]: `sizeof(uint8_t) = 1`. 
What does this "magic" operation `sizeof(uint8_t) * 2` mean?
Please, use named constants for such purposes

Comment (IvanPoliksenov) [reply inference-engine/src/vpu/graph_transformer/src/stages/exp_generateproposals.cpp]: The sizeof is used at the detection_output. Change to constants anyway?

**Comment by Maxim-Doronin** [comment]
@IvanPoliksenov, @elatkin, any updates on this?

**Review by elatkin — APPROVED** [review]

**Comment by elatkin** [comment]
> @IvanPoliksenov, @elatkin, any updates on this?

I am on vacations till end of Oct 2020. Please request @avershov to review

**Review by avershov — APPROVED** [review]

**Comment by elatkin** [comment]
Closing this PR as replaced with OpenVINO #[3093](https://github.com/openvinotoolkit/openvino/pull/3093)

**Pull request closed without merging**